types/selenium-webdriver/chrome.d.ts

```diff
@@ -5,7 +5,7 @@ import * as remote from "./remote";
 /**
  * Creates a new WebDriver client for Chrome.
  */
-export class Driver extends webdriver.ChromiumWebDriver {
+export class Driver extends webdriver.chromium.Driver {
```

Comment:
To be clear, is it expected that this name changes? This is a critical package, so I expect a lot of this kind of renaming to be a breaking change for downstream users.

Reply:
This class name is difficult to recall because it exclusively appears in the chromium.d.ts and is not utilized elsewhere. If that is the case, these other classes in the chrome.d.ts, edge.d.ts, and other files should be changed to `ChromeWebDriver` or `EdgeWebDriver` more consistently. Currently, they have been shortened to just the `Driver` name. To me, this change will move the source closer to the [upstream repository](https://github.com/SeleniumHQ/selenium/blob/67ba0054e34cff220e68b457e88a0859114ff250/javascript/node/selenium-webdriver/chromium.js#L602), making it more traceable going forward. I anticipate that it won’t cause any significant issues, since users will be testing it with Chrome by default.

Reply:
I don't mind if you and the owners think it won't be a problem, this is just a "critical" package so caution is always advisable.

Reply:
Resolved now. I reverted to the old name for compatibility.